src/components/DatasetEditorForm.vue

```diff
@@ -82,7 +82,7 @@
                     </v-card-text>
                     <v-container>
                         <v-text-field label="wis2box auth token for 'processes/wis2box'" v-model="token" rows="1"
-                            :append-icon="showToken ? 'mdi-eye' : 'mdi-eye-off'" :type="showToken ? 'text' : 'password'"
+                            :append-icon="showToken ? 'mdi-eye' : 'mdi-eye-off'" :type="showToken ? 'text' : 'password'" autocomplete="one-time-code"
```

Comment:
why one time code?

Reply:
There is an issue with Chrome attempting to autofill components based on its type being `password`. @maaikelimper has experienced various glitches thanks to Chrome's insistence. There's no official way to disable autofill, the closest we have found is this HTML tag `autocomplete`, which when set to `one-time-code` should make Chrome see it as a fresh password (like the OTP used in 2FA) and thus no autofill.